vertx-web/src/main/java/io/vertx/ext/web/handler/BodyHandler.java

```diff
@@ -75,6 +85,15 @@ static BodyHandler create() {
   static BodyHandler create(String uploadDirectory) {
     return new BodyHandlerImpl(uploadDirectory);
   }
+  
+  /**
+   * Set whether file uploads will be handled
+   * 
+   * @param handleFileUploads  true if they should be handled
+   * @return reference to this for fluency
```

Comment:
You could make it `<tt>this</tt>` to make it stand out in the text (for readability).

Reply:
I copy/pasted it from setUploadsDirectory and setMergeFormAttributes I'd rather prefer to leave it as the other javadoc comments are